src/Tests/testsProjectCannotRunOnHelixList.txt

```diff
@@ -0,0 +1,11 @@
+Microsoft.DotNet.MSBuildSdkResolver.Tests
```

Comment:
Question: what distinguishes between tests that can and can't be run in helix?

Reply:
literally what cannot run on helix. Some will fail in helix, I want to get most of the test running in helix to reduce the time first. Plus if they run in parallel, it won't add to overall test time. They are mostly old dotnet/cli tests and websdk tests. dotnet/cli tests have dependency on the repo structure. I haven't get time to untangle them.